app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -58,18 +50,10 @@
 import org.mozilla.focus.observer.AverageLoadTimeObserver;
 import org.mozilla.focus.open.OpenWithFragment;
 import org.mozilla.focus.popup.PopupUtils;
-import org.mozilla.focus.session.NullSession;
-import org.mozilla.focus.session.Session;
-import org.mozilla.focus.session.SessionCallbackProxy;
-import org.mozilla.focus.session.SessionManager;
-import org.mozilla.focus.session.Source;
+import org.mozilla.focus.session.*;
 import org.mozilla.focus.session.ui.SessionsSheetFragment;
 import org.mozilla.focus.telemetry.TelemetryWrapper;
-import org.mozilla.focus.utils.Browsers;
-import org.mozilla.focus.utils.Features;
-import org.mozilla.focus.utils.StatusBarUtils;
-import org.mozilla.focus.utils.SupportUtils;
-import org.mozilla.focus.utils.UrlUtils;
+import org.mozilla.focus.utils.*;
```

Comment:
\[Checkstyle\] ERROR: Using the '\.\*' form of import should be avoided \- org\.mozilla\.focus\.utils\.\*\.